cypress/support/config/services.js

```diff
@@ -25,20 +32,28 @@ export default {
           : {
               asset: 'c4vlle3q337o',
             },
+      nonExistentarticle:
+        Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
```

Comment:
Why dont we run the nonExistantArticle test on test?

Reply:
Similar reason https://github.com/bbc/simorgh/pull/2624#discussion_r305788440...befotre i consolidated the news and persian error pages into one test file the `errorpersianpage` was been described to run on local only using `describeForLocalOnly`

Reply:
It is currently difficult to see from the config which services have error pages locally, on Test and on Live. It would be better to simplify this so it is very clear. For example, there can be a `nonExistantArticle` value being 
```
nonExistantArticle: { asset: 'c0000000000o' }, 
```
for all the services, and there being an extra field for each service in this file: `errorPageEnvironments: ['local', 'test', 'live'],`. So, `news` will have all three environments, but `yoruba` will only have `errorPageEnvironments: ['local'],` at the moment.

Reply:
This is the way the other fields like articles, frontpage are being wriiten. To be consistent we need to change them throughout as well which will then mean we may have to update the other test files too accordingly.I would say we could raise it as a separate issue and make it consistent for all the fields in `services.js`

Reply:
@PriyaKR Sure, happy to fix forwards in a separate PR.